modules/ROOT/pages/install/get-started.adoc

```diff
@@ -171,13 +219,11 @@ NAME   TYPE     DATA   AGE
 csb    	Opaque       5      7m58s
 ----
 
-To access the couchbase cluster UI console: 
+To access the Couchbase Web Console:
 
 [source,console]
 ----
 $ kubectl port-forward couchbase-instance-winhhoku-0000 8091
 ----
 
-Go to http://localhost:8091 and login with username as Administrator & password as password.
-
-// The following example demonstrates how to deploy a Couchbase cluster on your local with Couchbase Service Broker from scratch.
+Go to `http://localhost:8091` and login with the credentials used when provisioning the Couchbase cluster service.
```

Comment:
For the record, this is a complete service broker anti-pattern.  If you want UI access you need to provision an ingress in your templates and return the endpoint address as part of the binding.